README.md

```diff
@@ -1,11 +1,12 @@
 
 <img src="Images/AnchorKit.png" alt="AnchorKit" />
 
-![Swift Version](https://img.shields.io/badge/swift-3.1-orange.svg?style=flat)
+![Swift Version](https://img.shields.io/badge/swift-4-orange.svg?style=flat)
+![Swift Version](https://img.shields.io/badge/swift-3-orange.svg?style=flat)
 [![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
 [![CocoaPods Compatible](https://img.shields.io/cocoapods/v/AnchorKit.svg)](https://img.shields.io/cocoapods/v/AnchorKit.svg)
 [![Platform](https://img.shields.io/cocoapods/p/AnchorKit.svg?style=flat)](http://cocoapods.org/pods/AnchorKit)
-[![Build Status](https://travis-ci.org/Weebly/AnchorKit.svg?branch=master)](https://travis-ci.org/Weebly/AnchorKit)
+[![Build Status](https://travis-ci.org/Weebly/AnchorKit.svg?branch=swift-4)](https://travis-ci.org/Weebly/AnchorKit)
```

Comment:
I believe you want this to remain `master`

Reply:
good catch lol